uc-crux-llvm/src/UCCrux/LLVM/Overrides/Check.hs

```diff
@@ -138,6 +144,20 @@ data SomeCheckedConstraint m sym (argTypes :: Ctx (FullType m)) =
   forall inTy atTy.
     SomeCheckedConstraint (CheckedConstraint m sym argTypes inTy atTy)
 
+data SomeCheckedConstraint' m =
+  forall sym argTypes inTy atTy.
+    SomeCheckedConstraint' (CheckedConstraint m sym argTypes inTy atTy)
+
+-- NOTE(lb): The explicit kind signature here is necessary for GHC 8.8/8.6
+-- compatibility.
+data CheckedCall m sym arch (argTypes :: Ctx (FullType m)) =
```

Comment:
What does this type represent? Is this recording all of the constraints checked at a given call site (in a given context)?

Reply:
Also, are these created inside the generated checking override? Do we strip the override from the call stack?

Reply:
Added comments.